.infra/common.ts

```diff
@@ -321,6 +321,10 @@ export const workers: Worker[] = [
     topic: 'api.v1.user-top-reader',
     subscription: 'api.user-top-reader-added',
   },
+  {
+    topic: 'kvasir.post-translated',
```

Comment:
do we want to do `v1`, just gives us some flexibility in the future if anything breaking changes

Reply:
Yeah, good idea.

Reply:
https://github.com/dailydotdev/streams/pull/43
```suggestion
    topic: 'kvasir.v1.post-translated',
```